corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -141,7 +142,7 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         });
 
         $('#select-case').off('click').click(function () {
-            FormplayerFrontend.trigger("menu:select", caseId);
+            FormplayerFrontend.trigger("menu:select", caseId, model.smartLinkParams);
```

Comment:
@Robert-Costello This general area is where you'd likely modify to change the Continue button (plus the HTML template and probably some changes to the view itself).

...Having said that, it probably makes sense to pull the comparison logic in `router.js` back into here, since you'll want the same logic to determine whether or not to change the button.

Reply:
Updated in 9648d524622